File: dev-superior-algoritmo-e-logica-de-programacao/c/exercicios/numeros_pares.c
#include <stdio.h>

int main(){
    int N, cont;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &N);

    int ver[N];

    for (int i = 0; i < N; i++) {
        printf("Digite um numero: ");
        scanf("%d", &ver[i]);
    }

    cont = 0;
    printf("\nNUMEROS PARES:\n");
    for (int i = 0; i < N; i++) {
        if (ver[i] % 2 == 0) {
            printf("%d  ", ver[i]);
            cont++;
        }
    }

    printf("\n\nQUANTIDADE DE PARES = %d\n", cont);

    return 0;
}

File: dev-superior-algoritmo-e-logica-de-programacao/java/src/exercicios/NumerosPares.java
package exercicios;

import java.util.Scanner;

public class NumerosPares {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		int N, cont;

	    System.out.print("Quantos numeros voce vai digitar? ");
		N = sc.nextInt();

	    int[] vet = new int[N];

		for (int i = 0; i < N; i++) {
	    	System.out.print("Digite um numero: ");
	        vet[i] = sc.nextInt();
	    }

		cont = 0;
		System.out.println("\nNUMEROS PARES:");
		for (int i = 0; i < N; i++) {
			if (vet[i] % 2 == 0) {
	        	System.out.printf("%d  ", vet[i]);
				cont++;
	        }
	    }

		System.out.printf("\n\nQUANTIDADE DE PARES = %d\n", cont);

		sc.close();
	}
}
